msg/SpawnEntityState.msg

```diff
@@ -0,0 +1,4 @@
+# Holds an entity state, xml description and UE tags
+string xml                  # Entity XML description as a string : entity type, etc
+ue_msgs/EntityState state   # Entity pose, twist, name, reference_frame
+string[] tags               # tags to use in UE
```

Comment:
Just a comment, `Spawn` prefix makes itself signified as a service/action, so given the msg contents, could it be named like `EntityInfo` for clarity?

Reply:
Yeah maybe `EntityInfo` is a better name